tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -377,6 +377,70 @@ def test_slurm_memory_based_scheduling(
     )
 
 
+@pytest.mark.usefixtures("region", "os", "instance", "scheduler")
+@pytest.mark.slurm_scontrol_reboot
+def test_scontrol_reboot(
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+    scheduler_commands_factory,
+):
+    cluster_config = pcluster_config_reader()
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    slurm_commands = scheduler_commands_factory(remote_command_executor)
+
+    # Clear clustermgtd logs before starting the tests
+    remote_command_executor.run_remote_command("sudo truncate -s 0 /var/log/parallelcluster/clustermgtd")
```

Comment:
why is this necessary?

Reply:
Because as of now static nodes are always marked as unhealthy at the beginning of the life of the cluster, during the first iteration of clustermgtd. So we have to remove those messages if we want the test to be meaningful.

Alternatively we could build a function that checks the logs only from a certain point in time - which is out of scope for the time being, and also probably quite complicated to generalize for all possible log files we might want to analyze.

Reply:
Agree, let's stick with the current implementation.